### PR TITLE
feat: add binding for sorting

### PR DESCRIPTION
**Pull request by ThunderFlight**

### CODE REVIEW

**Review by SlikeDollar — APPROVED** [review]

File: src/main.zig [hunk 1]
```diff
@@ -80,6 +80,18 @@ pub fn main() !void {
...
+                }
+
+                if (key.matches('j', .{}) and std.mem.eql(u8, last_keybind, "s")) {
+                    try sorting.sort(music, sorting.SortMethods.greater);
+                }
+
+                if (key.matches('k', .{}) and std.mem.eql(u8, last_keybind, "s")) {
+                    try sorting.sort(music, sorting.SortMethods.less);
+                }
+
```
Comment (VKTRenokh) [inline src/main.zig]: What will happen if i press j again after i pressed sj?

**Review by VKTRenokh — CHANGES_REQUESTED** [review]

File: src/keybinds/timeout.zig [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const std = @import("std");
+
+pub fn timeout_s(time_ms: u64, last_keybind: *[]const u8) void {
```
Comment (VKTRenokh) [inline src/keybinds/timeout.zig:3]: Rename this function please

File: src/main.zig [hunk 3]
```diff
@@ -57,7 +58,7 @@ pub fn main() !void {
     var playlist_paths: [1][]const u8 = .{try std.fs.path.join(allocator, &[2][]const u8{ home.?, ".config/cmus/playlists" })};
 
     const music = try playlists.getPlaylists(allocator, &playlist_paths);
-    try sorting.sort(music, sorting.SortMethods.greater);
+
     defer {
```
Comment (VKTRenokh) [inline src/main.zig:62]: return this line

File: src/misc/timeout.zig [hunk 4]
```diff
@@ -0,0 +1,6 @@
+const std = @import("std");
+
+pub fn timeout(time_ms: u64, last_keybind: *[]const u8) void {
+    std.time.sleep(time_ms);
+    last_keybind.* = "";
+}
```
Comment (VKTRenokh) [inline src/misc/timeout.zig]: Are you sure we need this file?

**Review by VKTRenokh — CHANGES_REQUESTED** [review]

**Review by VKTRenokh — APPROVED** [review]

**Review by SlikeDollar — APPROVED** [review]

**Pull request merged**